framework/auth/oauth_scopes.py

```diff
@@ -41,6 +41,10 @@ class CoreScopes(object):
     APPLICATIONS_READ = 'applications_read'
     APPLICATIONS_WRITE = 'applications_write'
 
+    NODE_LOG_READ = 'nodes.logs_read'
+
+    NULL = 'null'
```

Comment:
@icereval, @abought 

Is adding a NULL scope (for always read-only resources) a sane thing to do?